charts/kargo/values.yaml

```diff
@@ -150,6 +150,12 @@ api:
     ## you by your identity provider.
     clientID: <client ID>
 
+    ## The client ID for the OIDC client.
+    ## 
+    # Some OIDC providers (such as Dex) require a separate clientID for different
+    # callback URLs such as the web app vs. CLI http://localhost login.
```

Comment:
As with the regular `clientID` field above, could we add a note here that this field is ignored if the built-in Dex is enabled -- in which case it will be automatically configured?

Reply:
Non-blocking.